asapdiscovery-cli/asapdiscovery/cli/cli_args.py

```diff
@@ -87,6 +87,15 @@ def output_dir(func):
     )(func)
 
 
+def overwrite(func):
+    return click.option(
+        "--overwrite",
+        is_flag=True,
+        default=False,
```

Comment:
Does setting this to False reproduce the original behavior of the workflow?

Reply:
I guess I should have read the initial comment more closely.  I guess it does not, but I feel as though, in general, by default you should not overwrite, so this is likely better behavior.

Reply:
Yeah that's what I'm thinking. We're still in an early enough stage of development that I don't feel too bad about changing default behavior when justified!